server/src/test/menu.js

```diff
@@ -0,0 +1,64 @@
+import chai, { expect } from "chai";
+import chaiHttp from "chai-http";
+import app from "../app";
+
+chai.use(chaiHttp);
+chai.should();
+
+describe("menu", () => {
+  describe("GET Menu", () => {
+    it("should get the menu for the day", done => {
+      chai
+        .request(app)
+        .get("/api/v1/menu")
+        .end((err, res) => {
+          res.should.have.status(200);
+          res.body.should.be.a("object");
+          done();
+        });
+    });
+  });
+});
+
+describe("menu", () => {
+  describe("GET Menu", () => {
+    it("should get menu for the day", done => {
+      chai
+        .request(app)
+        .get("/api/v1/menu")
+        .end((err, res) => {
+          res.should.have.status(200);
+          res.body.should.be.a("object");
+          done();
+        });
+    });
+  });
+
+  describe("POST Menu", () => {
+    it("it should be able to POST the menu for the day ", done => {
+      chai
+        .request(app)
+        .post("/api/v1/menu")
+
+        .end((err, res) => {
+          res.should.have.status(400);
+          res.body.should.have.property("status");
```

Comment:
You want to write more robust assertions. Assert that the properties in the response.body object are exactly what they should be